example.init.yaml

```diff
@@ -1,37 +1,59 @@
+features:
```

Comment:
Could we derive this list in code and just use the labels as annotations in the YAML?

Reply:
i.e. if the user put `tls: true` then we know TLS is on and we have to create some certificates. Then if the user gave a secret for DigitalOcean DNS, we know we have to create the corresponding secret etc?

Reply:
Done this in latest commit 👍